i18n/fa/messages.fa.xlf

```diff
@@ -0,0 +1,5560 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+  <file source-language="en" datatype="plaintext" original="ng2.template" target-language="fa">
+    <body>
+      <trans-unit id="70cd6a0cff2699d1c553c292ae76f41bb3b8e0de" datatype="html">
+        <source>Edit a resource</source>
+        <target>ویرایش یک منبع</target>
```

Comment:
We have no reviewer for the translation. If you could find someone it would be great, possibly with Kubernetes experience.

Reply:
I don't know anybody for this but I'll tweet for that, maybe I can find somebody.

Reply:
You can look in the Kubernetes docs repo or on Slack.